setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+from glob import glob
+import os
+import sys
+
+from setuptools import setup, Extension
+from Cython.Build  import cythonize
+
+if sys.argv[-1] == 'setup.py':
+    print('To install, run \'python setup.py install\'')
+
+if sys.version_info[:2] < (2, 7):
+    print(
+        'networkx-lemon requires Python version 2.7 or later' +
+        ' ({}.{} detected).'.format(*sys.version_info[:2]))
+    sys.exit(-1)
+
+libraries = [
+    ('lemon', {'sources': glob('src/lemon/base.cc'),
+               'depends': glob('src/lemon/*.h') +
+                          glob('src/lemon/concepts/*.h') +
+                          glob('src/lemon/bits/*.h'),
+               'include_dirs': ['src']})]
+
+ext_modules = cythonize(
+    [Extension('', [''],
+               include_dirs=['src', 'src/lemon', 'src/lemon/bits',
+                             'src/lemon/concepts'],
```

Comment:
In the Cython files to write, we will also consistently include LEMON header files with the `lemon/` prefix. So `['src']` should suffice here, too.